code/datalab-app/api.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:7.10.0
+
+LABEL maintainer "joshua.foster@stfc.ac.uk"
+
+RUN mkdir -p /usr/src/api
+
+WORKDIR /usr/src/api
+
+COPY ./dist/api /usr/src/api
+
+COPY ./package.json /usr/src/api
+
+RUN npm install --production
+
+ENV PORT=8000
```

Comment:
I think that this default should live in the config in the code rather than in the docker file. See dynamic proxy for an example.